docs/changelog.md

```diff
@@ -14,9 +14,16 @@ permalink: /changelog/
   * Android (SDK: 28, Support Libraries: 28.0.0)
   * Kotlin (Stdlib: 1.3.10, Coroutines: 1.0.1)
   * GeckoView (Nightly: 66.0.20181217093726, Beta: 65.0.20181211223337, Release: 64.0.20181214004633)
-  * Mozilla App Services (FxA: 0.11.5, Sync Logins: 0.11.5, Places: 0.11.5)
+  * Mozilla App Services (FxA: 0.12.1, Sync Logins: 0.12.1, Places: 0.12.1)
   * Third Party Libs (Sentry: 1.7.14, Okhttp: 3.12.0)
 
+# 0.35.1
+
+* [Commits](https://github.com/mozilla-mobile/android-components/compare/v0.35.0...v0.35.1)
+
+* Compiled against:
+  * Mozilla App Services (FxA: 0.12.1, Sync Logins: 0.12.1, Places: 0.12.1)
```

Comment:
We usually keep the full list of dependencies and add this little arrows after the versions that have changed:  🔺

Reply:
Fixed. Thanks @csadilek